test/autorest/complexgroup/complexgroup_test.go

```diff
@@ -67,3 +75,85 @@ func TestPutValid(t *testing.T) {
 	}
 	deepEqualOrFatal(t, result, expected)
 }
+
+// TODO check this
+func TestGetInvalid(t *testing.T) {
+	client := getBasicClient(t)
+	_, err := client.GetInvalid(context.Background())
+	if err == nil {
+		t.Fatalf("GetInvalid expected an error")
+	}
+	// i, s := int(1), "abc"
+	// expected := &complexgroup.GetInvalidResponse{
+	// 	StatusCode: http.StatusOK,
+	// 	Basic:      &complexgroup.Basic{ID: &i, Name: &s},
+	// }
+	// deepEqualOrFatal(t, result, expected)
+}
+
+func TestGetEmpty(t *testing.T) {
+	client := getBasicClient(t)
+	result, err := client.GetEmpty(context.Background())
+	if err != nil {
+		t.Fatalf("GetEmpty: %v", err)
+	}
+	expected := &complexgroup.GetEmptyResponse{
+		StatusCode: http.StatusOK,
+		Basic:      &complexgroup.Basic{},
+	}
+	deepEqualOrFatal(t, result, expected)
+}
+
+// TODO nil or null?
+func TestGetNull(t *testing.T) {
```

Comment:
If the endpoint specifies Null will we rename everything to nil?

Reply:
Should come from the `operationId` in the swagger.

Reply:
Ok great, then I'm leaving it as GetNull since that's what the swagger specifies